pkg/container/templates/go.tmpl

```diff
@@ -37,18 +26,78 @@ ENV CGO_ENABLED=0 \
     GOARCH=amd64 \
     GO111MODULE=on
 
+# Set working directory
+WORKDIR /build
+
 {{if .IsLocalPath}}
 # Copy the local source code
-COPY . /app/
+COPY . /build/
+
+# Build the application
+RUN go build -o /app/mcp-server {{.MCPPackage}}
```

Comment:
when we arrive tat this point, have we sanitized the package name? or can it contain some malicious chars?

Reply:
I'm not sure, but we should. Can you open an issue for me to verify? This is probably something to fix outside of this pr.